package.json

```diff
@@ -42,14 +43,15 @@
     "fastboot-transform": "^0.1.3",
     "hammerjs": "^2.0.8",
     "matchmedia-polyfill": "^0.3.2",
+    "patch-package": "^6.5.1",
     "polyfill-nodelist-foreach": "^1.1.2",
     "propagating-hammerjs": "^2.0.1",
     "resolve": "^1.13.1",
     "sass": "^1.23.7",
     "tinycolor2": "^1.4.1"
   },
   "devDependencies": {
-    "@ember/jquery": "^1.1.0",
+    "@ember/jquery": "2.0.0",
```

Comment:
we should probably be removing `@ember/jquery`. Is there anything useful that it's actually being used for?